src/views/webview-app/connection-status.tsx

```diff
@@ -0,0 +1,166 @@
+
+const connectedContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[4],
+});
+
+const statusDotStyles = css({
+  width: spacing[2],
+  height: spacing[2],
+  borderRadius: '50%',
+  pointerEvents: 'none',
+  display: 'inline-block',
+  marginRight: spacing[2],
+});
+
+const connectedStatusDotStyles = cx(
+  statusDotStyles,
+  css({
+    backgroundColor: palette.green.base,
+  })
+);
+
+const disconnectedStatusDotStyles = cx(
+  statusDotStyles,
+  css({
+    backgroundColor: palette.red.base,
+  })
+);
+
+const statusContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[1],
+  alignItems: 'center',
+  justifyContent: 'center',
+  '@media(min-width: 500px)': {
+    '&': {
+      flexDirection: 'row',
+      gap: spacing[2],
+    },
+  },
+});
+
+const connectionNameStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const playgroundCtaContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[3],
+  alignItems: 'center',
+  '@media(min-width: 500px)': {
+    '&': {
+      gap: spacing[5],
+      justifyContent: 'center',
+      flexDirection: 'row',
+    },
+  },
+});
+
+const textContainerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[1],
+});
+
+const ctaTextStyles = css({
+  fontSize: '18px',
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const ConnectionStatusConnected: React.FC<{ connectionName: string }> = ({
+  connectionName,
+}) => {
+  return (
+    <div className={connectedContainerStyles}>
+      <div className={statusContainerStyles}>
+        <Body as="div">
+          <span className={connectedStatusDotStyles} />
+          Connected to:
+        </Body>
+        <Body as="div" className={connectionNameStyles}>
+          <strong>{connectionName}</strong>
+          <IconButton
+            onClick={renameActiveConnection}
+            aria-label="Rename connection"
+          >
+            <Icon size="xsmall" glyph="Edit" />
+          </IconButton>
+        </Body>
+      </div>
+      <div className={playgroundCtaContainerStyles}>
+        <Body className={textContainerStyles} as="div">
+          <div className={ctaTextStyles}>All set. Ready to start?</div>
+          <div className={ctaTextStyles}>
+            Create a playground.
+            <IconButton
+              target="_blank"
+              rel="noopener"
+              as="a"
+              href={LINKS.extensionDocs('playgrounds')}
+              aria-label="Learn about playgrounds"
+            >
+              <Icon glyph="InfoWithCircle" />
+            </IconButton>
+          </div>
+        </Body>
+        <Button
+          onClick={createNewPlayground}
+          variant="baseGreen"
+          aria-label="Create playground"
+        >
+          Create playground
+        </Button>
+      </div>
+    </div>
+  );
+};
+
+// eslint-disable-next-line react/no-multi-comp
```

Comment:
We could set this lint rule off globally, I think we do in Compass. Maybe this is something we get free when we update our lint to the shared one so feel free to ignore.